checker/client.go

```diff
@@ -34,7 +37,9 @@ func GetClients(ctx context.Context, repoURI, localURI string, logger *log.Logge
 	clients.VulnerabilitiesClient, // vulnClient
 	error,
 ) {
-	var githubRepo clients.Repo
+	var repo clients.Repo
```

Comment:
Could we ensure that the GitLab specific code in this file only gets triggered if `SCORECARD_EXPERIMENTAL` env var is set?